README.md

```diff
@@ -114,6 +114,33 @@ A value which has a Setoid must provide an `equals` method. The
 
 2. `equals` must return a boolean (`true` or `false`).
 
+### Ord
+
+A value that implements the Ord specification must also implement
+the [Setoid](#setoid) specification.
+
+1. `a.compare(a) <= 0` (reflexivity)
```

Comment:
I was simply copying the laws from PureScript. No other reason. `===` works just as well and is probably clearer.